lib/src/int64.dart

```diff
@@ -630,18 +633,16 @@ class Int64 implements IntX {
   }
 
   @override
-  List<int> toBytes() {
-    List<int> result = List<int>(8);
-    result[0] = _l & 0xff;
-    result[1] = (_l >> 8) & 0xff;
-    result[2] = ((_m << 6) & 0xfc) | ((_l >> 16) & 0x3f);
-    result[3] = (_m >> 2) & 0xff;
-    result[4] = (_m >> 10) & 0xff;
-    result[5] = ((_h << 4) & 0xf0) | ((_m >> 18) & 0xf);
-    result[6] = (_h >> 4) & 0xff;
-    result[7] = (_h >> 12) & 0xff;
-    return result;
-  }
+  List<int> toBytes() => [
```

Comment:
There is a question here as to whether the result should be a regular growable list, or a fixed-length or Uint8List or something else. It would be easier to change to a new type if the code has the same structure.

You could use the old code with
```
List<int> result = [0, 0, 0, 0, 0, 0, 0, 0];
```

Reply:
> You could use the old code with

I don't understand how that is meaningfully different from what is here. In either case the instance is a growable list of the same type.

If anything we'd use `List<int> result = List<int>.filled(8, 0);` so that it's not growable and then use the old code.

Reply:
I don't understand how that change would help things, but I could create a non-growable list filled with zeros. That is probably the better change as it was non-growable before.

Reply:
Went with that approach